sdlf-utils/pipeline-examples/datalake-workload-management/lambda/send-events-sqs/src/lambda_function.py

```diff
@@ -0,0 +1,147 @@
+import json, boto3, datetime
+import logging
+import os
+import urllib.parse
+from botocore.exceptions import ClientError
+import uuid
+import random
+
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+sns = boto3.client('sns')
```

Comment:
Usage of interface from datalake library.

Reply:
Changed the way repository is structured -- Now contain two sections. sdlf-wlm-integration now uses interfaces and configurations